python/paddle/fluid/layers/nn.py

```diff
@@ -4710,6 +4711,73 @@ def multiplex(inputs, index):
     return out
 
 
+def mask_extract(input, mask, name=None):
+    """
+    **Mask Extract Layer**
+
+    In the training of masked language model, one usual approach is masking some
+    input tokens at random, and predicting only those masked tokens. This layer
+    extracts the representations and indices of these masked tokens from an
+    entire input sequence and feeds them to some loss layer.
+
+    :attr:`mask` must have the same first dimension with :attr:`input`, in which
+    the valid indices indicate the chosen masked tokens. While the reset invalid
```

Comment:
Can you refine this sentence: `While the reset invalid indices, here including all the negative integers, stand for the unmasked tokens.`